content/en/docs/tasks/administer-cluster/kms-provider.md

```diff
@@ -11,11 +11,22 @@ This page shows how to configure a Key Management Service (KMS) provider and plu
 
 * {{< include "task-tutorial-prereqs.md" >}} {{< version-check >}}
 
+{{< tabs name="tab_with_md" >}}
+{{% tab name="v1" %}}
 * Kubernetes version 1.10.0 or later is required
 
 * etcd v3 or later is required
 
 {{< feature-state for_k8s_version="v1.12" state="beta" >}}
+{{< /tab >}}
+{{% tab name="v2" %}}
+* Kubernetes version 1.25.0 or later is required
+
```

Comment:
We need the feature gate set in `kube-apiserver` - `--feature-gates=KMSv2=true` to be able to configure a KMS v2 provider